src/ontology/uberon-edit.obo

```diff
@@ -80971,7 +80971,7 @@ relationship: adjacent_to UBERON:0002336 ! corpus callosum
 [Term]
 id: UBERON:0004053
 name: external male genitalia
-def: "The external masculine genital organs, including the penis and scrotum." [MP:0009199]
+def: "The external masculine genital organs. In mammals, this includes the penis and scrotum." [MP:0009199]
```

Comment:
I would advice to remove the second sentence, since there are several mammals that lack penis and/or scrotum. Also, I agree with Chris that a less mammalian biased definition would be more appropriate.

Reply:
I removed the second sentence.